reference_frame: "gps_origin"

control_action_interval: 0.01 # [s]
uvdar_msg_interval: 0.1 # [s]

# offset from the leader position to be maintained
desired_offset:
  x: -4.5 # [m]
  y: 4.5 # [m]
  z: 0.0 # [m]

# rotation offset around the world vertical axis
heading_offset: -1.22 # [rad]

# Noise parameters of the Kalman velocity estimator

# Process noise
Q: [8, 0, 0, 0, 0, 0,
    0, 8, 0, 0, 0, 0,
    0, 0, 8, 0, 0, 0,
    0, 0, 0, 2.5, 0, 0,
    0, 0, 0, 0, 2.5, 0,
    0, 0, 0, 0, 0, 2.5]

# # Measurement noise
R: [22.5, 0, 0,
    0, 22.5, 0,
    0, 0, 22.5]



## Parameters for trajectory approach with 1m/s velocity constraint
# Process noise
#Q: [8, 0, 0, 0, 0, 0,
#    0, 8, 0, 0, 0, 0,
#    0, 0, 8, 0, 0, 0,
#    0, 0, 0, 3, 0, 0,
#    0, 0, 0, 0, 3, 0,
#    0, 0, 0, 0, 0, 3]

# Measurement noise
#R: [22.5, 0, 0,
#    0, 22.5, 0,
#    0, 0, 22.5]